syft/pkg/cataloger/deb/parse_dpkg_status.go

```diff
@@ -6,12 +6,12 @@ import (
 	"fmt"
 	"io"
 	"regexp"
-	"strconv"
 	"strings"
 
 	"github.com/anchore/syft/internal"
-
+	"github.com/anchore/syft/internal/log"
 	"github.com/anchore/syft/syft/pkg"
+	"github.com/jonasagx/datasize"
```

Comment:
looks like this should be `github.com/c2h5oh/datasize`

Reply:
we use `github.com/dustin/go-humanize` for some of these cases, would that work instead of pulling in a new lib?

Reply:
it did work, thanks